packages/inputs-minimal/src/textfield/index.jsx

```diff
@@ -20,6 +20,7 @@ const TextField = ({
   disabled,
   onFocus,
   onBlur,
+  type,
```

Comment:
Maybe add a `Password` (I'm assuming that's what you're using it for) instead of having type here? The type attribute can change a lot about how textfields behave or look like. Let's say you are passing `number` here - would the attachments still work? You would pretty much need to check for every type there is. That's why I think it's better to have separate components instead of this `type` prop -> more control

Reply:
Maybe call the prop `__type` to mark it for internal use and then simply use it in the `Password` component?